config.go

```diff
@@ -97,11 +97,17 @@ func (l *Loader) parseStruct(ctx context.Context, ref *reflect.Value) error {
 
 		key := tag
 		var required bool
+		var bcknd string
 
 		if idx := strings.Index(tag, ","); idx != -1 {
 			key = tag[:idx]
-			if tag[idx+1:] == "required" {
-				required = true
+			opts := tag[idx+1:]
+
+			// Check if the first 8 chars correspond to the required option
+			required = opts[:8] == "required"
```

Comment:
This will panic with `config:"foo,req"`.
Options are comma separated, the tag format is basically this `config:"<name>[,<opt>]*"`
You just need to split the options and check each option individually.